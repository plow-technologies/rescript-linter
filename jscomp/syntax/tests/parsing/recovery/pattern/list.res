switch x {
| list{} => ()
| list{1, list{} => ()
| list{}...1, ...list{3, 4} => ()
}
