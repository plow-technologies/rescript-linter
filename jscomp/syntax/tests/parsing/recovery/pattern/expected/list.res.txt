
  Syntax error!
  tests/parsing/recovery/pattern/list.res:3:17-19

  1 │ switch x {
  2 │ | list{} => ()
  3 │ | list{1, list{} => ()
  4 │ | list{}...1, ...list{3, 4} => ()
  5 │ }

  Did you forget a `}` here?


  Syntax error!
  tests/parsing/recovery/pattern/list.res:4:9-11

  2 │ | list{} => ()
  3 │ | list{1, list{} => ()
  4 │ | list{}...1, ...list{3, 4} => ()
  5 │ }
  6 │ 

  Did you forget a `=>` here?


  Syntax error!
  tests/parsing/recovery/pattern/list.res:4:13

  2 │ | list{} => ()
  3 │ | list{1, list{} => ()
  4 │ | list{}...1, ...list{3, 4} => ()
  5 │ }
  6 │ 

  I'm not sure what to parse here when looking at ",".

;;match x with | [] -> () | 1::[]::[] -> () | [] -> 1
;;[3; 4]
;;()